security/README.md

```diff
@@ -17,6 +17,18 @@ override the defaults set at the API or service level.
 The code generated by goa makes the information needed to authorize available to
 the service methods.
 
+## Enabling the plugin
+
+Enable this plugin by importing its package with name `_` and its DSL package
```

Comment:
Maybe something like:
------
To enable the plugin and make use of the security DSL simply import both the `security` and the `dsl` packages as follows:

```go
import (
   _ "goa.design/plugins/security"
   . "goa.design/plugins/security/dsl"
)
```
Note the use of the blank identifier to import the `security` package which is necessary as the package is imported solely for its side-effects (initialization).
-----
is a bit clearer?